### PR TITLE
update benchmark results markdown

### PR DESCRIPTION
**Pull request by WongGawa**
Thank you for your contribution to the MindYOLO repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindyolo/blob/master/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,15 @@
...
 
-MindYOLO is [MindSpore Lab](https://github.com/mindspore-lab)'s software toolbox that implements state-of-the-art YOLO series algorithms, [support list and benchmark](MODEL_ZOO.md). It is written in Python and powered by the [MindSpore](https://mindspore.cn/) AI framework.
-
-The master branch supporting **MindSpore 2.0/2.1**.
+MindYOLO implements state-of-the-art YOLO series algorithms based on MindSpore.
+The following is the corresponding `mindyolo` versions and supported `mindspore` versions.
+|  mindyolo  |  mindspore  |  tested hardware
+|    :--:    |     :--:    |     :--:       |
+|   master   |    master   |   ascend 910*  | 
+|    0.4     |    2.3.0    |   ascend 910*  |
```
Comment (Ash-Lee233) [inline README.md:20]: ascend 910* &  ascend 910
两种都有测试结果

File: benchmark_results.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# MindYOLO Benchmark and Baselines
+
+## Detection
+<details open markdown>
+<summary><b>performance tested on Ascend 910A(8p) with graph mode</b></summary>
```
Comment (Ash-Lee233) [inline benchmark_results.md:5]: 910(8p)

File: benchmark_results.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+| YOLOX  | M                  |   8 * 8   |    640    | MS COCO 2017 |    46.7     | 25.3M  | [yaml](./configs/yolox/yolox-m.yaml)         | [weights](https://download.mindspore.cn/toolkits/mindyolo/yolox/yolox-m_300e_map467-1db321ee.ckpt)                  |
+| YOLOX  | L                  |   8 * 8   |    640    | MS COCO 2017 |    49.2     | 54.2M  | [yaml](./configs/yolox/yolox-l.yaml)         | [weights](https://download.mindspore.cn/toolkits/mindyolo/yolox/yolox-l_300e_map492-52a4ab80.ckpt)                  |
+| YOLOX  | X                  |   8 * 8   |    640    | MS COCO 2017 |    51.6     | 99.1M  | [yaml](./configs/yolox/yolox-x.yaml)         | [weights](https://download.mindspore.cn/toolkits/mindyolo/yolox/yolox-x_300e_map516-52216d90.ckpt)                  |
+| YOLOX  | Darknet53          |   8 * 8   |    640    | MS COCO 2017 |    47.7     | 63.7M  | [yaml](./configs/yolox/yolox-darknet53.yaml) | [weights](https://download.mindspore.cn/toolkits/mindyolo/yolox/yolox-darknet53_300e_map477-b5fcaba9.ckpt)          |
+</details>
+
+<details open markdown>
+<summary><b>performance tested on Ascend 910B(8p)</b></summary>
```
Comment (Ash-Lee233) [inline benchmark_results.md:35]: 910*(8p)

File: README.md (hunk 1 above)
Comment (WongGawa) [reply README.md]: Done

File: benchmark_results.md (hunk 2 above)
Comment (WongGawa) [reply benchmark_results.md]: Done

File: benchmark_results.md (hunk 3 above)
Comment (WongGawa) [reply benchmark_results.md]: Done

**Review by vigo999 — APPROVED** [review]

**Pull request merged**